packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md

```diff
@@ -0,0 +1,7 @@
+* **Redirect URI** &#8212; Provide one or more login redirect URIs where Okta sends the OAuth responses. The URIs must be absolute URIs. You must add at least one redirect URI.
+
+* **Optional: Initiate Login URI** &#8212; Provide a URI if you want the Okta integration to handle redirecting your users to your app to start the sign-in request. When the end users click an Okta app, they are redirected to the `initiate_login_uri` of the client application, which constructs an authorization request and redirects the end user back to Okta.
```

Comment:
"When the end users click an Okta app, they are": I'm a bit confused by this. Is it when the end user clicks a tile in the Okta dashboard?

Reply:
Yes. Changed to:
"When end users click a tile in their Okta dashboard, they are..."